.circleci/config.yml

```diff
@@ -16,26 +16,34 @@ jobs:
       - image: web3f/ci-commons:v2.1.3
     steps:
       - checkout
+      - run:
+          name: Import external alertrules
```

Comment:
just to double check: is this necessary even if it is already present in importAndTestPrometheusRule job ?